src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java

```diff
@@ -51,17 +51,19 @@ public CaseDocument bundle(
                 bundleFilename,
                 caseData
             );
-
+        log.info("calling stitching api end point for " + caseData.getLegacyCaseReference());
```

Comment:
logger should be in this format 
            log.info(String.format("Http Status %s ", e.status()), e); .. appending is not a good way of adding logs.

Reply:
made changes